### ISSUE TITLE
Font families should be mainly select based

### ISSUE THREAD

**Issue by jorgefilipecosta** [ItsJonQ/g2#163]
The font family control on the typography tools https://g2-components.xyz/iframe.html?id=designtools-presentation-typographypanel--default&viewMode=story should be mainly select based.
The user will be presented with a set of font-family options. The user can choose one of the predefined options. I guess if a value outside of one of the options is passed we may render a text input, but that should not be the default.

**Comment by aristath** [comment]
Agreed. It could be a-la-select2 which would accept custom values too, but it should primarily work as a select input :+1:

**Comment by ItsJonQ** [comment]
Sounds like we need a [ComboBox](https://github.com/ItsJonQ/g2/issues/74) :)

That's what `react-select` is (at least it's primary configuration is a combobox).

We can use [downshift](https://www.downshift-js.com/use-combobox), which is what we did for the recently added [`SelectDropdown`](https://github.com/ItsJonQ/g2/tree/master/packages/components/src/select-dropdown) (same library used in `@wordpress/components`).
